src/ua_securechannel.h

```diff
@@ -77,6 +77,7 @@ struct UA_SecureChannel {
     UA_MessageSecurityMode  securityMode;
     UA_ChannelSecurityToken securityToken; /* the channelId is contained in the securityToken */
```

Comment:
Add a few comments that explain the securityTokens and how they are rotated.
This will save the next developer a few hours of tracking things down.